app/jobs/submit_symphony_request_job.rb

```diff
@@ -3,14 +3,28 @@
 class SubmitSymphonyRequestJob < ActiveJob::Base
   queue_as :default
 
-  def perform(request, options = {})
+  def perform(request_id, options = {})
```

Comment:
Hmm, maybe this is my naiveté (given I haven't worked much with background jobs) but I was under the impression that with `ActiveJob` we didn't have to pass the ID any longer as long as the object that we're passing into the job implements `GlobalID` (which `ActiveRecord` does).

I'm basing this off the [GlobalID section in the ActiveJob Rails Guide](http://edgeguides.rubyonrails.org/active_job_basics.html#globalid).  Was that not working for you in the requests app?